src/components/Authors.js

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prop-types */
+import React from 'react';
```

Comment:
- [ ] Let's try not to disable Linter checks. Linter checks help us write clean, better, and professional code and help us become better developers. Disabling Linters defies its purpose and hence, can't help you become a better and professional developer. Please remove the ESLint disable and try to fix the error 👍🏽